ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -1690,6 +1700,7 @@ static void zend_jit_stop_reuse_ip(void)
 static int zend_jit_interrupt_handler_stub(dasm_State **Dst)
 {
 	|->interrupt_handler:
+	|	ENDBR
```

Comment:
only few of the ``ENDBR`` inserted in stubs below are really necessary. Most of them just a helper functions that can't be called indirectly. Only counters need to be updated.

Reply:
Done.